### PR TITLE
issue 11: space-after-type-colon does not apply to the return type

### PR DESCRIPTION
**Pull request by johnsonw**
Given the following configuration:

```json
"flowtype/space-after-type-colon": [
   1,
   "never"
]
```

and a function:

```js
function fn (key: string): Object {
```

Running the linter will produce the following:

> xxx:yy  warning  There must be no space after "key" parameter type annotation colon  flowtype/space-after-type-colon

This patch will address this issue and ensure that the return type is also taken into consideration when checking for spaces after the colon.

### CODE REVIEW

File: src/rules/spaceAfterTypeColon.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import _ from 'lodash';
+import {format} from 'util';
```
Comment (gajus) [inline src/rules/spaceAfterTypeColon.js]: Please remove `format` in favour of string concatenation (as it was).

Your are welcome to submit a separate PR promising to use `util.format`, though this addition is not required to solve this issue.

**Comment by johnsonw** [comment]
Anything else I need to address in this patch for it to land? I can open a separate patch to handle space-before-type-colon.

**Comment by gajus** [comment]
@johnsonw Please squash the commits. If you do not know how, here is a guide. http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by johnsonw** [comment]
Commits have been squashed.

**Comment by gajus** [comment]
Thanks. Thats a lot clearer.

**Comment by gajus** [comment]
v2.2.5 has been published.
